layouts/taxonomy/category.terms.html

```diff
@@ -12,7 +12,7 @@
 
         <ul class="posts">
             <header>
-                <h1>{{ .Data.Plural }}</h1>
+                <h1>{{ i18n .Data.Plural }}</h1>
```

Comment:
NOTE: This is my website and **categories** was replaced by **destinations**, however I didn't change the behavior.

**Before:**
![image](https://user-images.githubusercontent.com/18215453/35472502-3c72a556-0368-11e8-953f-e127ec87e57b.png)

**After adding this change:**
![image](https://user-images.githubusercontent.com/18215453/35472516-787f07e2-0368-11e8-8878-66108f9e9dc4.png)

Should I add fall-back for this as well?